packages/squidex/schema/crn/schemas/research-outputs.json

```diff
@@ -76,6 +76,29 @@
           "isHalfWidth": false
         }
       },
+      {
```

Comment:
I think this is a legitimate case for an 'enum' type in Squidex, which is basically a regular string with allowed-values defined but has a `createEnum` option set to true and - what is important - all values conform to a certain pattern (basically no spaces and special chars). 

Have a look at how it was done here for example:
https://github.com/yldio/asap-hub/blob/208274e9de73beb7296e9b34b90e38b224124299/packages/squidex/schema/gp2/schemas/projects.json#L71-L98

Once we have it correctly defined as an enum (you might find it difficult to change this in an existing app in Squidex - in such case just create a new PR or make changes to existing PR app) you should refresh your graphql schema. You will now be able parse it more nicely